### ISSUE TITLE
Aria-label is missing from buttons

### ISSUE THREAD

**Issue by notPlancha** [notPlancha/Volume-Profiles-Spicetify-Extension-v2#18]
I forgot to add the aria label. Should be fairly easy to fix, but it should need Tippy afaik.
